.github/workflows/wheels.yml

```diff
@@ -202,12 +202,10 @@ jobs:
       - name: Install deps
         run: pip install -U setuptools-rust wheel build
       - name: Build sdist
```

Comment:
We're actually not building an sdist here not that it matters. Apparently we're only uploading a wheel for retworkx.